### PR TITLE
[Data History Explorer]: Table tab

### PR DESCRIPTION
**Pull request by iremgokce**
### Summary of changes 🔍 
- Added `TimeSeriesTable` tab components for the TimeSeriesViewer parent component to flat list all the time series data with **download** command bar option. Download action will download the series in ADX table columns (<Timestamp, Id, Key. Value>)
- I can share the command bar component between Chart and Table tab if we want to control the time picker and share query options in Table tab as well.

image
image
image

### Testing 🧪
Check it out in `TimeSeriesTable `internal component or in `DataHistoryExplorerModal` main parent component stories.

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [x] Added relevant labels
- [x] Requested developer reviews
- [x] Request UI review from design / PM
- [x] Verify all code checks are passing

### CODE REVIEW

File: src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesTable/TimeSeriesTable.styles.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            { overflow: 'auto', position: 'relative', flexGrow: 1 }
+        ],
+        notSetCell: [classNames.notSetCell, { fontStyle: 'italic' }],
+        subComponentStyles: {
+            loadingSpinner: { root: { height: '100%' } },
+            detailsList: {
+                root: { overflow: 'hidden', '.ms-DetailsRow': { fontSize: 13 } }
+            },
+            seriesColumn: (props) => {
+                return mergeStyleSets({
```
Comment (pasanchMSFT) [inline src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesTable/TimeSeriesTable.styles.ts:37]: why do you need mergeStyleSets here?

**Review by pasanchMSFT — APPROVED** [review]

**Review by darsney — APPROVED** [review]
LGTM, would be great to get MockAdapter to feature scale tests, and have stable stories.  Otherwise sweetttt!

File: src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesTable/TimeSeriesTable.styles.ts (hunk 1 above)
Comment (iremgokce) [reply src/Components/DataHistoryExplorer/Internal/TimeSeriesViewer/Internal/TimeSeriesTable/TimeSeriesTable.styles.ts:37]: I need to return a IProcessedStyleSet here to access its **root** what assigning it as a className as you also used in other places like [here](https://github.com/microsoft/iot-cardboard-js/blob/800150615c09ea53410ffcd1c1b0ec4d004db2e6/src/Components/ColorPills/ColorPills.types.ts#L46)

**Comment by iremgokce** [comment]
> LGTM, would be great to get MockAdapter to feature scale tests, and have stable stories. Otherwise sweetttt!

I will do stress tests and stable stories in a separate PR. First I want to merge all the features in. Error handling and linking to Property Inspector is next one.

**Pull request merged**